napalm_ios/ios.py

```diff
@@ -27,6 +27,7 @@
 from napalm_base.exceptions import ReplaceConfigException, MergeConfigException
 from napalm_base.utils import py23_compat
 import napalm_base.constants as C
+import napalm_base.helpers
```

Comment:
Okay, so @ebeahan can you convert this over to direct use of the helpers.mac() and then it should be ready to go.